images/iscsi-target-tgtd/README.md

```diff
@@ -0,0 +1,74 @@
+# Purpose
+
+This image can be used to export a file from a volume as an
+iSCSI lun - served by tgtd.
+
+The container has two "modes":
+
+1. Demo - The lun is prepoulated with a demo image (qemu demo image)
+2. Persist - The lun is pointing to an empty file of a given size
+   which resides on a persistent volume).
+
+
+# Build
+
+Easy:
+
+```bash
+$ docker build -t kubevirt/iscsi-target-tgtd .
+```
+
+# Demo flow
+
+The demo flow will download a demo image into the container,
+which is then exposed as a LUN.
+
+```bash
+# Docker
+$ docker run \
+  -p 3260:3260 \
+  -e GENERATE_DEMO_OS_SEED=true -it iscsi-target-tgtd
+
+
+# To test:
+# In another terminal
+$ qemu-system-x86_64 \
+  -drive file=iscsi://127.0.0.1/iqn.2017-01.io.kubevirt:sn.42/1
+
+# Or just to discover
+$ iscsiadm --mode discovery -t sendtargets --portal 127.0.0.1
+```
+
+# Persistent flow
```

Comment:
Is this flow so relevant for our testing on the VM? I think we can reduce this section to one sentence like this:

The luns are all served by images inside `/volume`. After a pod restart the content will be reset. If you want to keep it between restarts, back the `/volume` path in the Pod with a persistent volume.